content/docs/components/menu/theming.mdx

```diff
@@ -2,3 +2,332 @@
 id: menu
 scope: theming
 ---
+
+The `Menu` component is a multipart component. The styling needs to be applied
+to each part specifically.
+
+> To learn more about styling multipart components, visit the
+> [Component Style](/docs/styled-system/component-style#styling-multipart-components)
+> page.
+
+## Anatomy
+
+<ol type='A'>
+  <li>`button`</li>
+  <li>`list`</li>
+  <li>`item`</li>
+  <li>`groupTitle`</li>
+  <li>`command`</li>
+  <li>`divider`</li>
+</ol>
```

Comment:
Could you turn these into bullet points? I've realized the left padding looks pretty weird when using `<ol>`, so I updated the Input theming doc to fix that.